mix.exs

```diff
@@ -84,6 +84,10 @@ defmodule Kwtool.MixProject do
         "credo --strict",
         "sobelow --config"
       ],
+      "lint.fix": [
```

Comment:
Maybe `codebase.fix` to be consistent to existing one 👍 

And for `mix codebase`, maybe we need to add a check so that it fails on CI -> `"cmd ./assets/node_modules/.bin/stylelint --color ./assets/css"`

Reply:
Yes, i updated.